osu.Game/Overlays/Profile/Sections/PaginatedProfileSubsection.cs

```diff
@@ -169,5 +184,21 @@ protected override void Dispose(bool isDisposing)
             loadCancellation?.Cancel();
             base.Dispose(isDisposing);
         }
+
+        private partial class OnScreenIndicator : Component
+        {
+            private readonly Action subsectionOnScreen;
+
+            public OnScreenIndicator(Action subsectionOnScreen)
```

Comment:
Make an `init` property, else the naming doesn't make sense. Could also drop `subsection` from the name as the class itself says nothing about subsections.

Reply:
Addressed in https://github.com/ppy/osu/pull/33647/commits/35faee5647e22397d3a941f2ad9aab1b5391cae9 as well